Editorial review of the undeprecated traversal framework for 5.0 (#46)
neo4j/docs-java-reference: modules/ROOT/pages/java-embedded/traversal.adoc

output += "Number of friends found: " + numberOfFriends + "\n";
----

> It should give this output:

Reviewer: This is a little vague, perhaps this is more confident?
```suggestion
This will return the following output:
```

Number of friends found: 4
----

> === Who coded the Matrix?
> To find the master mind behind "The Matrix", follow all outgoing relationships which have the type `KNOWS` or `CODED_BY`.

Reviewer: This rhetorical question in the title looks a little odd - perhaps something like this as an alternative? (Also, mastermind is one word I think!)

```suggestion
=== Solving the Matrix

To find the mastermind behind "The Matrix", follow all outgoing relationships which have the type `KNOWS` or `CODED_BY`.
```

}
----

> This should be the output:

Reviewer: Again, this sounds a little hopeful, and we can be more confident:

```suggestion
This will return the following output:
```

[[examples-uniqueness-of-paths-in-traversals]]
== Uniqueness of Paths in traversals

> The follow example demonstrates the use of node uniqueness.

Reviewer 2: The following example?
